src/cdk/testing/webdriver/webdriver-element.ts

```diff
@@ -0,0 +1,263 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {
+  _getTextWithExcludedElements,
+  ElementDimensions,
+  EventData,
+  ModifierKeys,
+  TestElement,
+  TestKey,
+  TextOptions
+} from '@angular/cdk/testing';
+import {Button, By, Key, WebElement} from 'selenium-webdriver';
+
+/**
+ * Maps the `TestKey` constants to WebDriver's `Key` constants.
+ * See https://github.com/SeleniumHQ/selenium/blob/trunk/javascript/webdriver/key.js#L29
+ */
+const keyMap = {
+  [TestKey.BACKSPACE]: Key.BACK_SPACE,
+  [TestKey.TAB]: Key.TAB,
+  [TestKey.ENTER]: Key.ENTER,
+  [TestKey.SHIFT]: Key.SHIFT,
+  [TestKey.CONTROL]: Key.CONTROL,
+  [TestKey.ALT]: Key.ALT,
+  [TestKey.ESCAPE]: Key.ESCAPE,
+  [TestKey.PAGE_UP]: Key.PAGE_UP,
+  [TestKey.PAGE_DOWN]: Key.PAGE_DOWN,
+  [TestKey.END]: Key.END,
+  [TestKey.HOME]: Key.HOME,
+  [TestKey.LEFT_ARROW]: Key.ARROW_LEFT,
+  [TestKey.UP_ARROW]: Key.ARROW_UP,
+  [TestKey.RIGHT_ARROW]: Key.ARROW_RIGHT,
+  [TestKey.DOWN_ARROW]: Key.ARROW_DOWN,
+  [TestKey.INSERT]: Key.INSERT,
+  [TestKey.DELETE]: Key.DELETE,
+  [TestKey.F1]: Key.F1,
+  [TestKey.F2]: Key.F2,
+  [TestKey.F3]: Key.F3,
+  [TestKey.F4]: Key.F4,
+  [TestKey.F5]: Key.F5,
+  [TestKey.F6]: Key.F6,
+  [TestKey.F7]: Key.F7,
+  [TestKey.F8]: Key.F8,
+  [TestKey.F9]: Key.F9,
+  [TestKey.F10]: Key.F10,
+  [TestKey.F11]: Key.F11,
+  [TestKey.F12]: Key.F12,
+  [TestKey.META]: Key.META
+};
+
+/** Converts a `ModifierKeys` object to a list of Protractor `Key`s. */
+function toWebDriverModifierKeys(modifiers: ModifierKeys): string[] {
+  const result: string[] = [];
+  if (modifiers.control) {
+    result.push(Key.CONTROL);
+  }
+  if (modifiers.alt) {
+    result.push(Key.ALT);
+  }
+  if (modifiers.shift) {
+    result.push(Key.SHIFT);
+  }
+  if (modifiers.meta) {
+    result.push(Key.META);
+  }
+  return result;
+}
+
+/** A `TestElement` implementation for WebDriver. */
+export class WebdriverElement implements TestElement {
```

Comment:
```suggestion
export class WebDriverElement implements TestElement {
```
It looks it's normally capitalized like "WebDriver"